maker/src/contract_setup.rs

```diff
@@ -19,15 +19,15 @@ use maia_core::PartyParams;
 use model::olivia::Announcement;
 use model::Dlc;
 use model::Identity;
-use model::Order;
+use model::Offer;
 use model::Role;
 use model::Usd;
 use xtra::prelude::MessageChannel;
 use xtra_productivity::xtra_productivity;
 use xtras::SendAsyncSafe;
 
 pub struct Actor {
-    order: Order,
+    order: Offer,
```

Comment:
should this be `offer`? 

```suggestion
   offer: Offer,
```

Reply:
Yes and no 😅 
This is legacy code and I think no matter how much I try to change it, it will always be odd because the boundaries are just not right. I agree that there is quite a lot of weirdness in this file because of renaming `Order` to `Offer`. I am not sure if it's a good idea to try to clean this up if we remove it soon.

Reply:
Given that this is the only instance where we literally have a direct conflict I'll change that one :)